packages/oasis/oasis.0.4.9/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://forge.ocamlcore.org/frs/download.php/1690/oasis-0.4.9.tar.gz"
```

Comment:
It looks like the forge.ocamlcore.org has a TLS certificate issue, preventing installation on some distros:

```
/ # curl -OL https://forge.ocamlcore.org/frs/download.php/1690/oasis-0.4.9.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.haxx.se/docs/sslcerts.html

```